galaxy.yml

```diff
@@ -9,6 +9,14 @@ description: A variety of Ansible content to help automate the management of AWS
 license_file: COPYING
 tags: [amazon, aws, cloud]
 repository: https://github.com/ansible-collections/amazon.aws
+dependencies:
+  "ansible.windows": "*"
```

Comment:
These dependencies are only needed for he integration tests. Is there a way to only add them in the tests/integration/requirements.yml file only?

Reply:
This should not appear here, the collection does not have any dependencies with these collections.
This is needed for test purposes only.

Reply:
From my understanding, collections specified in tests/integration/requirements.yml are not installed by Zuul. If we remove these collections from the required-projects list in Zuul, we need to ensure they are added here to ensure the tests pass on Zuul.

Reply:
We have relied on the required-projects list to install the collections used in our tests. Going forward, there's a possibility that the ansible.netcommon project might be removed from Zuul, as its tests no longer utilize Zuul for CI.

Reply:
> From my understanding, collections specified in tests/integration/requirements.yml are not installed by Zuul.

Then, IMO, before we update the Zuul configs, that's what we should be fixing.  There **is** a need to separate the run time dependencies from the test dependencies.  Just because we install something to make our lives easier when building our test cases shouldn't mean we force customers to install them for run-time it's bad practice.